main.go

```diff
@@ -167,7 +168,15 @@ func runDaemon(cmd *cobra.Command) error {
 		fmt.Printf("pouchd version: %s, build: %s, build at: %s\n", version.Version, version.GitCommit, version.BuildTime)
 		return nil
 	}
-	metrics.EngineVersion.WithLabelValues(version.GitCommit).Set(1)
+
+	kernelVersion, err := kernel.GetKernelVersion()
+	if err != nil {
+		return fmt.Errorf("failed to get kernel version: %s", err)
+	}
+	metrics.EngineVersion.WithLabelValues(
```

Comment:
do we need add a test case for this？

Reply:
I just realized this problem and will add the test.